solr/core/src/java/org/apache/solr/cloud/ConfigSetCmds.java

```diff
@@ -70,12 +70,9 @@ private static NamedList<Object> getConfigSetProperties(
       throws IOException {
     byte[] oldPropsData = configSetService.downloadFileFromConfig(configName, propertyPath);
     if (oldPropsData != null) {
-      InputStreamReader reader =
-          new InputStreamReader(new ByteArrayInputStream(oldPropsData), StandardCharsets.UTF_8);
-      try {
+      try (InputStreamReader reader =
```

Comment:
As you change code, use of "var" in lines like this would use probably one less line yet still be quite clear.
Do or not as you wish.

Reply:
will keep it in mind for the future